### PR TITLE
Implement favicon support

### PR DESCRIPTION
**Pull request by li-kai**
Generated from https://realfavicongenerator.net/

Opted to not use .ico for now. Browser support is there, and we don't even plan to support IE. Saves us about 10kb.

### CODE REVIEW

File: v3/webpack/webpack.config.dev.js [hunk 1]
```diff
@@ -39,17 +40,18 @@ const developmentConfig = merge([
...
+      new webpack.WatchIgnorePlugin([parts.PATHS.node, parts.PATHS.build]),
       // Enable multi-pass compilation for enhanced performance
       // in larger projects. Good default.
       // Waiting on: https://github.com/jantimon/html-webpack-plugin/issues/533
-      new webpack.HotModuleReplacementPlugin({
-        // multiStep: true,
-      }),
+      new webpack.HotModuleReplacementPlugin(
+        {
+          // multiStep: true,
```
Comment (yangshun) [inline v3/webpack/webpack.config.dev.js]: Is this still needed? If not can remove?

File: v3/webpack/webpack.config.prod.js [hunk 2]
```diff
@@ -28,6 +29,13 @@ const productionConfig = merge([
         title: 'NUSMods',
         template: path.join(parts.PATHS.app, 'index.html'),
       }),
+      // Copy files from static folder over to dist
+      new CopyWebpackPlugin(
+        [
+          { from: 'static' },
+        ],
+        { copyUnmodified: true },
```
Comment (yangshun) [inline v3/webpack/webpack.config.prod.js]: Trailing commas in a function call is a syntax error in node < 8. Let's do without it if we can.

File: v3/webpack/webpack.config.dev.js (hunk 1 above)
Comment (li-kai) [inline v3/webpack/webpack.config.dev.js]: Yep still needed, issue not fixed yet.

**Comment by yangshun** [comment]
:shipit:

**Pull request merged**